tests/instrumentation/asyncio/aiohttp_client_tests.py

```diff
@@ -38,10 +39,14 @@
 pytestmark = [pytest.mark.asyncio, pytest.mark.aiohttp]
 
 
-async def test_http_get(instrument, event_loop, elasticapm_client, waiting_httpserver):
+@pytest.mark.parametrize("use_yarl", [True, False])
```

Comment:
Great use of `parametrize`, much better than my suggestion of copying the test :+1: